cypress/support/step-definitions/fieldset-steps.js

```diff
@@ -0,0 +1,17 @@
+import { legendInput, legendPreview } from '../../locators/legend';
+
+When('I set legend to {string}', (legend) => {
+  legendInput().clear().type(legend);
+});
+
+When('I set legend to empty', () => {
+  legendInput().clear();
```

Comment:
what if we will use previous method and input " space " as a parameter? in this case `I set legend to empty` will be unnecessary.

Reply:
'space' is not empty character